task5/src/main/java/com/clouway/core/FundsRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.clouway.core;
+
+/**
+ * Created by Kristiyan Petkov  <kristiqn.l.petkov@gmail.com> on 09.06.16.
+ */
+public interface FundsRepository {
+
+  void deposit(double amount, String email);
```

Comment:
should be deposit(amount, userId); In this case email is the userId and it fits but usually is the id in DB. - Only for information